src/MatBlazor/Components/MatLink/BaseMatLink.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace MatBlazor
+{
+    public class BaseMatLink: BaseMatDomComponent
+    {
+        protected async override Task OnFirstAfterRenderAsync()
+        {
+            await base.OnFirstAfterRenderAsync();
+            await JsInvokeAsync<object>("matBlazor.matButton.init", Ref);
+        }
+
+        public BaseMatLink()
+        {
+            ClassMapper
+                .Add("mdc-button")
+                .Add("mdc-link")
+                .If("mdc-button--raised", () => this.Raised)
+                .If("mdc-button--unelevated", () => this.Unelevated)
+                .If("mdc-button--outlined", () => this.Outlined)
+                .If("mdc-button--dense", () => this.Dense);
+        }
+
+        /// <summary>
+        ///  Event occurs when the user clicks on an element.
+        /// </summary>
+        [Parameter]
+        public EventCallback<MouseEventArgs> OnClick { get; set; }
+
+        /// <summary>
+        /// Stop propagation of the OnClick event
+        /// </summary>
+        [Parameter]
+        public bool OnClickStopPropagation { get; set; }
+
+        [Parameter]
+        public ICommand Command { get; set; }
+
+        /// <summary>
+        ///  Command parameter.
```

Comment:
Same here

Reply:
I have removed the space before ```Command```. Please let me know if you meant something else.